### ISSUE TITLE
Docker development environment

### ISSUE THREAD

**Issue by bowdena** [rubyaustralia/melbourne-ruby#225]
Not an expert by any means.

Now that rails 7.1 ships by default with a production docker environment (https://github.com/rails/rails/blob/f0d433bb46ac233ec7fd7fae48f458978908d905/guides/source/7_1_release_notes.md), I got interested in trying to dockerise my development environment on the mac m1 architecture.

I'm happy to talk about two alternate approaches to this specifically for mac architecture and much of the information will be cross platform applicable if anyone is interested.

**Comment by ZimbiX** [comment]
Good idea. I've done a bunch of Dockerisation work, and we had Dockerised development (and prod) environments for all our microservices at GreenSync.

**Comment by simran-sawhney** [comment]
hey @bowdena double checking if you are still happy to present it this month at meetup ?

**Comment by simran-sawhney** [comment]
hey hey @bowdena - do you wanna run this as lightning talk in May 23rd ?
